### ISSUE TITLE
NTR: Breast biopsy: myxoma  (Lab: pathology: biopsies & morphology)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#9144]
New term request Breast biopsy: myxoma  (elthat_100318030249):

**Comment by pnrobinson** [comment]
@MickeySegal
Adding new term: 
- ID: HP:6000671
- label: Breast myxoma
- parents: [Neoplasm of the breast (HP:0100013)](https://hpo.jax.org/app/browse/term/HP:0100013)
- pmid: PMID:17214801
- def: A rare, benign mesenchymal tumor of the breast characterized histologically by stellate, round, and spindle-shaped cells with vesicular nuclei and scant cytoplasm that are scattered in the myxoid stroma.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/9144
- synonyms:
